### ISSUE TITLE
Disallow putting invalid values in the grid size selector

### ISSUE THREAD

**Issue by getkey** [getkey/ble#24]
![image](https://user-images.githubusercontent.com/6908397/98026088-e70fa300-1e0a-11eb-9c46-7429357057cf.png)
![image](https://user-images.githubusercontent.com/6908397/98026183-060e3500-1e0b-11eb-8656-517cb17ffafd.png)

What should be disallowed:
- Negative values
- Floating values

Context & original bug report: https://discord.com/channels/526188679006388254/601609930486841365/772917528170594304

**Comment by mse-k** [comment]
wdym floating values, you mean numbers with stuff after the decimal point because that actually works even if it being drawn is buggy

**Comment by getkey** [comment]
Yes floating-point is just the programming way to say numbers decimal-point.

And yeah actually there is no reason to disallow floating-point grid, it just needs to be fixed visually, so I opened #25. :slightly_smiling_face:

**Comment by vooxal** [comment]
I would just run abs value on update or something, maybe not the most efficient solution but a solution nevertheless

**Comment by getkey** [comment]
Nah, it's just a matter of setting the [min attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/number#min) to the `<input>`.
